### DISCUSSION TITLE
🐛 Bugs / Potential Bugs Question Discussion

### DISCUSSION THREAD

**Discussion by pwdel** [openpredictionmarkets/socialpredict#332] [General]
This is a discussion thread for any bugs found or questions on potential bugs found in the SocialPredict platform.

https://github.com/openpredictionmarkets/socialpredict

We are sorry for any potential confusion or frustration any bugs may have caused you as a user or admin of the platform. Please let us know if you're seeing anything strange or have questions. Bugs might be technical in nature, or might be considered, "this just isn't presented as well as it could be," from a user experience perspective.

Please abide by our code of conduct in these discussions. https://github.com/openpredictionmarkets/socialpredict/blob/main/CODE_OF_CONDUCT.md

**Comment by astrosnat** [comment]
**Protocol errors when running `go test -cover`**

Fresh install of go 1.23.1 on WSL2, fresh clone of `reportingstats` branch of repo.

When running `go test -cover` in `stats` folder receive protocol errors:

![Screenshot 2024-09-20 182506](https://github.com/user-attachments/assets/a7aae86e-45ca-447a-a0d6-b1c690dae02f)

I'm not using a proxy or VPN, but sometimes my internet is slow. Oddly enough, waiting 10 minutes and trying again fixed it?

**Reply by pwdel** [reply]
What I'm seeing here is basically, network latency. This can happen when downloading packages or repositories of any type, or linux/debian versions, whatever.

Yeah basically the solution is to try again. Sometimes it's on your end, sometimes it's on their end. The fact that you, "sometimes," have network problems tells me it was likely on your end. Even if you just had latency for a couple minutes, there could have been a timeout. So yep, trying again was the solution.

**Comment by astrosnat** [comment]
**Where credits should go, says "Failed to fetch user credit"**

![Screenshot from 2024-09-24 12-55-19](https://github.com/user-attachments/assets/9fc751fb-4b99-478d-bb0a-8cc93c83c3c8)

This is a known issue upon first spinning up a SocialPredict deployment. I was using the branch from [PR 341](https://github.com/openpredictionmarkets/socialpredict/pull/341).

Running on `prod`on Ubuntu 22.04.05 with Docker preinstalled (it's on a Ubuntu Docker Droplet)

Accessed website using Firefox 130.0 for Ubuntu 22.04

Edit: happened again when I logged in as a new user

![Screenshot from 2024-09-24 13-05-21](https://github.com/user-attachments/assets/b810fb47-85a0-4843-8a36-376e94340752)
